elasticsearch/Dockerfile.centos7

```diff
@@ -39,8 +39,11 @@ LABEL io.k8s.description="Elasticsearch container for EFK aggregated logging sto
 ADD elasticsearch.repo /etc/yum.repos.d/elasticsearch.repo
 # install the RPMs in a separate step so it can be cached
 RUN rpm --import https://packages.elastic.co/GPG-KEY-elasticsearch && \
+    yum install -y centos-release-scl && \
+    yum-config-manager --enable rhel-server-rhscl-7-rpms && \
```

Comment:
really? rhel-server-rhscl-7-rpms from a centos repo?

Reply:
https://www.softwarecollections.org/en/scls/rhscl/rh-maven33/

Reply:
It looks like one or the other:
```
# 1. Install a package with repository for your system:
# On CentOS, install package centos-release-scl available in CentOS repository:
$ sudo yum install centos-release-scl

# On RHEL, enable RHSCL repository for you system:
$ sudo yum-config-manager --enable rhel-server-rhscl-7-rpms
```
which to me means that since this is for CentOS, you only need to `yum install centos-release-scl` - I guess we'll need to do `yum-config-manager --enable rhel-server-rhscl-7-rpms` in Dockerfile for RHEL downstream.